.github/scripts/components-scripts/conformance-state.gcp.firestore-setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+# Set variables for GitHub Actions
+echo "GCP_PROJECT_ID=$GCP_PROJECT" >> $GITHUB_ENV
```

Comment:
is $GCP_PROJECT actually set at this point?

Reply:
This env var is set by the **Authenticate to Google Cloud** step.